GoogleDataTransport/GDTCORLibrary/GDTCORPlatform.m

```diff
@@ -241,12 +241,8 @@ - (void)endBackgroundTask:(GDTCORBackgroundIdentifier)bgID {
 #pragma mark - App environment helpers
 
 - (BOOL)isAppExtension {
```

Comment:
Should we use GoogleUtilities's isAppExtension? https://github.com/firebase/firebase-ios-sdk/blob/master/GoogleUtilities/Environment/third_party/GULAppEnvironmentUtil.m#L245

Or should we update GoogleUtilities to follow this as well?

Reply:
> Should we use GoogleUtilities's isAppExtension?

We cannot have a dep on GoogleUtilities